vendor/modules.txt

```diff
@@ -111,10 +111,10 @@ golang.org/x/crypto/ed25519/internal/edwards25519
 golang.org/x/crypto/pkcs12/internal/rc2
 # golang.org/x/net v0.0.0-20190415214537-1da14a5a36f2
 golang.org/x/net/idna
+golang.org/x/net/context
```

Comment:
Is this diff intentional?

Reply:
It is not; it seems to be spontaneously produced by some quirk of my local tooling. I'll revert it.